.travis.yml

```diff
@@ -1,58 +1,51 @@
 language: python
-python: 2.7
 sudo: false
+
 cache:
   pip: true
   directories:
     - $HOME/buildout-cache
+
 before_install:
-  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - mkdir $HOME/.buildout
-  - echo "[buildout]" > $HOME/.buildout/default.cfg
-  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
-  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
-  - pip install --upgrade pip setuptools zc.buildout
-  - if [[ `npm -v` != 3.10.7 ]]; then npm i -g npm@3.10.7; fi
-  - npm install -g grunt-cli
-
-env:
-  matrix:
```

Comment:
@loechel this test matrix runs tests against multiple versions of Plone and for the different package types. It is a two-dimensional matrix and running those against different Python versions would mean we end up with a three-dimensional matrix. The tox environment only contains the Python version as a parameter. Therefore we do not run tests against multiple Plone versions and package types any longer as far as I can see.